libs/authorizenet/driver/src/errors/auth-failed.ts

```diff
@@ -0,0 +1,12 @@
+import { DaffError, DaffInheritableError } from '@daffodil/core';
+
+/**
+ * An error thrown when either the API Login ID or Public Client Key values are incorrect.
+ */
+export class DaffAuthorizeNetAuthFailedError extends DaffInheritableError implements DaffError {
```

Comment:
What's the difference between this error and `DaffAuthorizeNetInvalidClientKeyError` and `DaffAuthorizeNetInvalidLoginIdError`?

Reply:
To be honest, I'm not completely sure. This is one of the ones that might be able to be condensed. I'm referencing [this page](https://developer.authorize.net/api/reference/features/acceptjs.html) for the error codes and there are 3 distinct errors.

I think the main difference is that the other errors will be thrown for missing keys whereas this one is just for invalid ones. I'm not totally sure about that however.

Reply:
oh ok, yea that sounds fine then